src/System.Net.Sockets/src/System/Net/Sockets/TCPClient.cs

```diff
@@ -6,77 +6,67 @@
-    /// <para>The <see cref='System.Net.Sockets.TcpClient'/> class provide TCP services at a higher level
-    ///    of abstraction than the <see cref='System.Net.Sockets.Socket'/> class. <see cref='System.Net.Sockets.TcpClient'/>
-    ///    is used to create a Client connection to a remote host.</para>
-    /// </devdoc>
+    // The System.Net.Sockets.TcpClient class provide TCP services at a higher level
+    // of abstraction than the System.Net.Sockets.Socket class. System.Net.Sockets.TcpClient
+    // is used to create a Client connection to a remote host.
     public class TcpClient : IDisposable
     {
         private Socket _clientSocket;
         private bool _active;
         private NetworkStream _dataStream;
 
-        //
-        // IPv6: Maintain address family for the client
-        //
+        // IPv6: Maintain address family for the client.
         private AddressFamily _family = AddressFamily.InterNetwork;
 
-        // specify local IP and port
-        /// <devdoc>
-        ///    <para>
-        ///       Initializes a new instance of the <see cref='System.Net.Sockets.TcpClient'/>
-        ///       class with the specified end point.
-        ///    </para>
-        /// </devdoc>
+        // Initializes a new instance of the System.Net.Sockets.TcpClient
+        // class with the specified end point.
         public TcpClient(IPEndPoint localEP)
         {
-            if (Logging.On) Logging.Enter(Logging.Sockets, this, "TcpClient", localEP);
+            if (Logging.On)
+            {
+                 Logging.Enter(Logging.Sockets, this, "TcpClient", localEP);
+            }
+
             if (localEP == null)
             {
                 throw new ArgumentNullException("localEP");
             }
-            //
+
             // IPv6: Establish address family before creating a socket
-            //
             _family = localEP.AddressFamily;
 
             initialize();
             Client.Bind(localEP);
-            if (Logging.On) Logging.Exit(Logging.Sockets, this, "TcpClient", "");
+            if (Logging.On)
+            {
+                 Logging.Exit(Logging.Sockets, this, "TcpClient", "");
+            }
         }
 
-        // TcpClient(IPaddress localaddr); // port is arbitrary
-        // TcpClient(int outgoingPort); // local IP is arbitrary
-
-        // address+port is arbitrary
-        /// <devdoc>
-        ///    <para>
-        ///       Initializes a new instance of the <see cref='System.Net.Sockets.TcpClient'/> class.
-        ///    </para>
-        /// </devdoc>
-        public TcpClient() : this(AddressFamily.InterNetwork)
+        // Initializes a new instance of the System.Net.Sockets.TcpClient class.
+        public TcpClient()
+            : this(AddressFamily.InterNetwork)
         {
-            if (Logging.On) Logging.Enter(Logging.Sockets, this, "TcpClient", null);
-            if (Logging.On) Logging.Exit(Logging.Sockets, this, "TcpClient", null);
+            if (Logging.On)
```

Comment:
Nit: This logging is redundant. 
On the other hand, removing it would break compat with public Desktop logging so I'm not sure if we should remove it.